test/cases/worker-executor.sh

```diff
@@ -151,8 +151,14 @@ subprocessPIDs+=( $! )
 ssh -oStrictHostKeyChecking=no -i "$KEYPAIR" "fedora@$EXECUTOR_IP" sudo dnf install -y osbuild-composer osbuild
 
 # no internet access during the build
+# TODO [thozza]: while debugging the test case, it turned out that the worker executor instance in fact has Internet access!
```

Comment:
@croissanne I didn't spend any time on this thing, because it was unrelated to the test case failure. I'd be happy to fix it, but I wanted to first check with you if you have any ideas because you've spent much more time working with this AWS feature.

Reply:
I've created https://github.com/osbuild/osbuild-composer/issues/4586